src/types/aws/ap/registration.ts

```diff
@@ -133,9 +133,9 @@ export type DoneContact = Append<
   ContactDetails
 >;
 export type DoneOrgDetails = Append<DoneContact, OrgDetails, {}>;
-export type DoneFSAInquiry = Append<DoneOrgDetails, FSAInquiry, {}>;
-export type DoneDocs = Append<DoneFSAInquiry, TDocumentation, {}>;
-export type DoneBanking = Append<DoneDocs, BankingDetails, {}>;
+export type DidFSAInquiry = Append<DoneOrgDetails, FSAInquiry, {}>;
```

Comment:
rename from `Done` to `Did` as user might have `"did"` previously, but no guarantee it's complete (might have been reset)